### DISCUSSION TITLE
Sort table with multi column sorting

### DISCUSSION THREAD

**Discussion by themahdavi** [filamentphp/filament#2208] [Ideas]
Hi everyone
I think if we can sort table with multi column is very usefull

for example i have products table and i need to sort records first by name (ASC) and second by count of buys (DESC) .

in version 2.11.3 doesn't support multi column sorting

**Comment by danharrin** [comment]
`->sortable(['count_of_buys', 'name'])`

**Reply by themahdavi** [reply]
where is this function to use ?
in each column or in form schema?
![image](https://user-images.githubusercontent.com/31320104/165224925-ddcef8ab-3530-4c46-a3a0-d2252c6e114c.png)

**Reply by danharrin** [reply]
Just on the column that you want the multi-sort to apply to

**Reply by themahdavi** [reply]
not working for this config
![image](https://user-images.githubusercontent.com/31320104/165594880-0d7644a6-207f-4018-a037-95e54d3caad5.png)
